src/chat/ChatScreen.tsx

```diff
@@ -111,8 +116,18 @@ export const ChatScreen: React.FC<ChatScreenProps> = ({
       );
 
       await firebaseInstance.sendMessage(messages.text);
+
+      timeoutMessageRef.current = setTimeout(() => {
+        sendNotificationChat?.();
+      }, timeoutSendNotify);
```

Comment:
I don't get why we have to wait for a while before sending push noti for that message but not instantly. Am I missing any requirements?

Reply:
this help project when apply this package can send notification by used API. When user not in Chatscreen will get notification

Reply:
I guess you forgot to cancel the push notification timeout when partners read the message within `timeoutSendNotify`. Please have a check @huydosgtech